packages/react-dev-utils/__tests__/getCSSModuleLocalIdent.test.js

```diff
@@ -11,7 +11,11 @@ const getCSSModuleLocalIdent = require('../getCSSModuleLocalIdent');
 
 const rootContext = '/path';
 const defaultClassName = 'class';
-const options = { context: undefined, hashPrefix: '', regExp: null };
+const defaultOptions = {
+  context: undefined,
+  hashSalt: undefined,
+  regExp: null,
+};
```

Comment:
With ccs-loader v6 / webpack v5 hashPrefix was removed in favor of hashSalt. hashSalt is undefined by default.